doc/adrs/130_add_more_filter_capabilities_on_diagram.adoc

```diff
@@ -0,0 +1,50 @@
+= ADR-127 - Add more filter capabilities on diagram
+
+== Context
+
+Right now, we have severals tools to filter diagram elements.
+
+=== Nodes
+
+We can hide or fade a node.
+
+In the case where the node is a compartiments, we can collapse/uncollapse. 
+
+We can also specify in the view dsl if we want to show icons label.
+
+=== Edges
+
+We can hide or fade an edge.
+
+We can also specify in the view dsl if we want to show icons label.
+
+== Decision
+
+We will add the capability to show/hide a label or an icon label on a diagram representation.
+
+Since this will bring new combinations of elements visible, hidden or partially hidden on a diagram, we will add a new representation to apply a specific view modifier to all selected elements on a diagram.
```

Comment:
- View modifier is specifically hidden or faded
- You want to say that we will let end users control details of the appearance of nodes and edges in the diagram since it's more than the view modifier